Include/longobject.h

```diff
@@ -212,6 +212,7 @@ PyAPI_FUNC(PyObject *) _PyLong_GCD(PyObject *, PyObject *);
 #ifndef Py_LIMITED_API
 PyAPI_FUNC(PyObject *) _PyLong_Rshift(PyObject *, size_t);
 PyAPI_FUNC(PyObject *) _PyLong_Lshift(PyObject *, size_t);
+PyAPI_FUNC(PyObject *) _PyLong_AddInt(PyLongObject *, int);
```

Comment:
Any reason not to name similar to the `INT_ADD` opcode? `_PyLong_IntAdd` instead of `AddInt`

Reply:
I think that the name of the API function is fine. but that of the instruction should change. The int is the right hand operand,
so `ADD_INT` seems more logical.

Reply:
Yeah just thinking if they can be consistent that would be great.